release-content/0.15/release-notes/15800_Add_mesh_picking_backend_and_MeshRayCast_system_parameter.md

```diff
@@ -1,4 +1,117 @@
 <!-- Add mesh picking backend and `MeshRayCast` system parameter -->
 <!-- https://github.com/bevyengine/bevy/pull/15800 -->
 
-<!-- TODO -->
+![A collection of geometric shapes, with a pointer showing a point on a hovered mesh. The indicator is perpendicular to the surface.](mesh_picking.png)
```

Comment:
It's a shame that mesh picking is called out further down as being disabled by default and discouraged due to performance concerns, but then the title image for this feature is explicitly mesh picking. I don't think there's anything that can be done about it though. These are real concerns, and mesh picking is the best visual representation of the feature.